### ISSUE TITLE
IDE glitches with OpenJDK

### ISSUE THREAD

**Issue by ascherer** [processing/processing#5135]
Running Processing 3.3.4 with OpenJDK 8u131-b11-0ubuntu1.16.04.2 gives the following problems:

* <del>Code 28-14 produces only two lines of output in the console window, but no scrollbar</del>
* Code 26-05: While playing with this code and separating the classes into individual files, I experienced some problems editing code in new tabs: sometimes the first column disappeared and the indenting wasn't quite correct – possibly related to copy/paste. — Reopening the project makes this problem disappear.

This is a provisional list of problems in this particular setting, separated from the more general issue #5111 and the related issue #4415. I'm trying to collect concrete examples where Processing does not work as well with OpenJDK as with Oracle Java.

_To be continued …_

**Comment by GKFX** [comment]
The console issue is a duplicate of #5110, now fixed.
Could you clarify exactly what you mean by the first column disappearing (the line numbering column? the first character of each line of text?)
Also what exactly was the indent issue? Was it related to what you see when you initially type in code, pasting in code, or the `ctrl`+`t` autoformatter? And what code showed the issue?

**Comment by ascherer** [comment]
@GKFX Thanks for pointing out #5110. After editing preferences.txt, code 28-14 works as expected with OpenJDK.

The problem with the editor has not occurred again. AFAIR, it happened in the _second_ tab while entering code for the `Ball` class of code 26-05 manually. `class` appeared as `lass`, and when I had finished the complete input, I had to strip one `c` from `cclass`, indent all lines by two spaces, and finally reformat the whole class definition. Maybe it was triggered by copy/pasting or some other ’control’ character, but as yet I couldn't reproduce the situation.
